src/main/webapp/components/directives/cache/configurationviewer.js

```diff
@@ -0,0 +1,53 @@
+(function () {
+  'use strict';
+
+  var module = angular.module('ispn.directives.cache.cacheconfiguration', ['ispn.services.utils']);
+
+  module.directive('cacheConfiguration', ['utils', function (utils) {
+      return {
+        restrict: 'E',
+        scope: {
+          currentCache: '=',
+          metadata: '=',
+          initDefaults: '=',
+          readOnly: '=',
+          cacheType: '@'
+        },
+        replace: false,
+        templateUrl: function(elem, attrs) {
+          return 'components/directives/cache/'.concat(attrs.cacheType).concat('.html');
```

Comment:
Instead of having multiple html files, one for each cache type, I would consider creating only one file (and it will be template of this directive) that will have ng-ifs in itself, to choose which tabs are shown depending on the type of cache.